templates/angular/igx-ts/combo/default/files/src/app/__path__/__name__.component.ts

```diff
@@ -0,0 +1,18 @@
+import { ChangeDetectorRef, Component, OnInit, ViewChild } from '@angular/core';
+import { IgxComboComponent  } from 'igniteui-angular';
+import { localData  } from "./local-data";
+
+@Component({
+	selector: 'app-$(filePrefix)',
+  	templateUrl: './$(filePrefix).component.html',
+  	styleUrls: ['./$(filePrefix).component.css']
+})
+
+export class $(ClassName)Component implements OnInit {
+
+	public lData = localData;
```

Comment:
You can still name that property `localData` if you want or you know - `states` or w/e.